sdcm/nemesis.py

```diff
@@ -761,18 +761,23 @@ def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC *
         InfoEvent(message='StartEvent - Adding new node to cluster').publish()
         new_node = self.cluster.add_nodes(
             count=1, dc_idx=self.target_node.dc_idx, enable_auto_bootstrap=True, rack=rack)[0]
-        self.monitoring_set.reconfigure_scylla_monitoring()
-        self.set_current_running_nemesis(node=new_node)  # prevent to run nemesis on new node when running in parallel
-        new_node.replacement_node_ip = old_node_ip
-        try:
-            self.cluster.wait_for_init(node_list=[new_node], timeout=timeout, check_node_health=False)
-            self.cluster.clean_replacement_node_ip(new_node)
-        except (NodeSetupFailed, NodeSetupTimeout):
-            self.log.warning("TestConfig of the '%s' failed, removing it from list of nodes" % new_node)
-            self.cluster.nodes.remove(new_node)
-            self.log.warning("Node will not be terminated. Please terminate manually!!!")
-            raise
-        self.cluster.wait_for_nodes_up_and_normal(nodes=[new_node])
+        with EventsSeverityChangerFilter(new_severity=Severity.WARNING,
```

Comment:
I'm not sure this is something we want.
If node joins and somehow it fails to load schema, we want see this error.

Reply:
the idea was to reduce the severity of the issue and not just ignore it

Reply:
But we ignore warnings most of  the time and we call it SUCCESS.
So, this is not a good enough solution, it's too risky to miss issues.

Reply:
in the case of a real issue, won't this schema mismatch continue to appear after the node joined the cluster?
what would you suggest?

Reply:
I don't know.
We should get an answer from Shlomi or a developer who knows this area if this is good enough to ignore schema mismatch during any node init.